WordGame/ViewController.swift

```diff
@@ -7,13 +7,129 @@
 
 import UIKit
 
-class ViewController: UIViewController {
-
+class ViewController: UITableViewController {
+    var allWords = [String]()
+    var usedWords = [String]()
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        navigationItem.leftBarButtonItem = UIBarButtonItem(barButtonSystemItem: .refresh, target: self, action: #selector(startGame))
+        navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .add, target: self, action: #selector(promptForAnswer))
+        if let startWordsURL = Bundle.main.url(forResource: "start", withExtension: "txt") {
+            if let startWords = try? String(contentsOf: startWordsURL){
+                allWords = startWords.components(separatedBy: "\n")
+            }
+        }
+        if allWords.isEmpty {
+            allWords = ["fileError"]
+        }	
+        startGame()
+    }
+    
+   @objc private func startGame() {
+        title = allWords.randomElement()
+        usedWords.removeAll(keepingCapacity: true)
+        tableView.reloadData()
+    }
+    
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return usedWords.count
+    }
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "Word", for: indexPath)
+        cell.textLabel?.text = usedWords[indexPath.row]
+        return cell
+    }
+    
+    @objc private func promptForAnswer () {
+        let alertController = UIAlertController(title: "Enter answer", message: nil, preferredStyle: .alert)
+        alertController.addTextField()
+        
+        let submitAction = UIAlertAction(title: "submit", style: .default) {
+            [weak self, weak alertController] action in
+            guard let answer = alertController?.textFields?[0].text else { return }
+            self?.submit(answer)
+        }
+        alertController.addAction(submitAction)
+        present(alertController, animated: true)
+    }
+    
+    private func submit (_ answer: String) {
+        let lowerAnswer = answer.lowercased()
+        if isPossible(word: lowerAnswer){
+            if isOriginal(word: lowerAnswer){
```

Comment:
Very minor: styling - Add Space between condition and bracket `{`
```suggestion
            if isOriginal(word: lowerAnswer) {
```

Reply:
No problem, I will fix it.